src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -438,7 +438,7 @@ CACHES = {
 }
 
 ELASTICSEARCH_HOSTS = [
-    {% for es_host in groups.elasticsearch -%}
+    {% for es_host in groups.es_coordinator -%}
```

Comment:
I think we need to be mindful of third party hosters here. To make it backwards compatible and to make the `es_coordinator` group optional, I would suggest that we use `groups.es_coordinator` if it is present and not empty, and otherwise we use `groups.elasticsearch` here.

Reply:
@dannyroberts made changes, please suggest if any changes are required. thanks